Products/ExternalEditor/ExternalEditor.py

```diff
@@ -130,7 +131,7 @@ def index_html(self, REQUEST, RESPONSE, path=None):
             if callable(title):
                 title = title()
             if isinstance(title, types.UnicodeType):
```

Comment:
`types.UnicodeType` does not exist on Python 3. And the code after it looks kind of strange to me.
I guess this will work:
```
if isinstance(title, six.text_type):
    title = title.encode('utf-8')
```

Reply:
This is a problem in PR #8 as well.

Reply:
Good catch, thank you! Btw, I noticed sixer skipped those also in `Products.Archetypes.Registry` (which we might not do this time) and `ply.lex` (which we probably need to)